contracts/SnappAuction.sol

```diff
@@ -8,6 +8,16 @@ contract SnappAuction is SnappBase {
     uint16 public constant AUCTION_BATCH_SIZE = 1000;
     uint8 public constant AUCTION_RESERVED_ACCOUNTS = 50;
     uint8 public constant AUCTION_RESERVED_ACCOUNT_BATCH_SIZE = 10;
+    
+    struct StandingOrders {
```

Comment:
```suggestion
    struct StandingOrder {
```

Reply:
Since the hash contains several orders, I like the plural

Reply:
Oh yes. right